src/stores/depositCreditCardStore.ts

```diff
@@ -0,0 +1,78 @@
+import {action, observable} from 'mobx';
+import {RootStore} from '.';
+import {DepositCreditCardApi} from '../api/depositCreditCardApi';
+import {ApiResponse} from '../api/types';
+import {convertFieldName, DepositCreditCardModel} from '../models/index';
+
+export class DepositCreditCardStore {
+  @observable defaultDeposit: DepositCreditCardModel;
+  @observable newDeposit: DepositCreditCardModel;
+  @observable gatewayUrls: {failUrl: string; okUrl: string; paymentUrl: string};
+
+  constructor(
+    readonly rootStore: RootStore,
+    private api?: DepositCreditCardApi
+  ) {
+    this.defaultDeposit = new DepositCreditCardModel();
+    this.newDeposit = new DepositCreditCardModel();
```

Comment:
i'm not sure it's a good idea to hold a local state of the deposit in the store. It brings a lot of pain in the transfers :) here is teh link https://github.com/reactjs/redux/issues/1287#issuecomment-175351978

can be improved later